### PR TITLE
fix broken atb integration tests on Xcode 11.4

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/414709148257752/1174879355199843
Tech Design URL:
CC:

**Description**:

Clear the ATB when launching from atb-tests

**Steps to test this PR**:
1. Using latest Xcode run the ATB integration tests
1. Launch the app and browse regularly to make sure there's no regressions

**Orientation Testing**:

* [x] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE
* [x] iPhone 8
* [x] iPhone X
* [ ] iPad

**OS Testing**:

* [ ] iOS 10
* [ ] iOS 11
* [x] iOS 12
* [x] iOS 13

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by subsymbolic** [comment]
I see a consistent failure `Neither element nor any descendant has keyboard focus` when running ` testWhenUserSearchesWithOldAtbThenAtbIsUpdated`. The same happens on BitRise.

error

Device: iPhone 8 emulator, OS 13.4.1

File: DuckDuckGo/AppDelegate.swift [hunk 1]
```diff
@@ -43,6 +43,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // MARK: lifecycle
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+                
+        if ProcessInfo.processInfo.environment.contains(where: { $0.key == "atb-testing" }) {
```
Comment (subsymbolic) [inline DuckDuckGo/AppDelegate.swift]: Optional: Can we extract this block into self documenting methods something like `ifAtbTesting` for the boolean check and `resetStatisticsData` for processing if that's true.

**Comment by brindy** [comment]
I made that change and tested it on a bunch of simulators on two different Macs and didn't have any problems.  I also created a new workflow and set it to use Xcode 11.4 and it's working there too, see https://app.bitrise.io/build/2e178e49153a7826#?tab=log

**Comment by subsymbolic** [comment]
@brindy it consistently fails for me on an iPhone 8 running 13.41.1 emulator (but is fine on others). Did you try that specific emulator? If CI passes again and tests in the description are all ok for you then it might be an issue on my system. Feel free to tick the tests and I'll approve the PR. This isn't user facing and if any issues turn up you can fix them then.

**Review by subsymbolic — APPROVED** [review]

**Review by subsymbolic — APPROVED** [review]
Looks good Brindy! And working on the iPhone 8 emulator now.

**Comment by brindy** [comment]
Thanks!

**Pull request merged**